src/conda_package_handling/__init__.py

```diff
@@ -1,3 +1,6 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+from . import _version
+__version__ = _version.get_versions()['version']
```

Comment:
Those lines are a duplication ... see lines 1-3.

Reply:
Good catch, this was versioneer updating the file automatically, d'oh.